### DISCUSSION TITLE
About the zoom level of acquired data

### DISCUSSION THREAD

**Discussion by tarukuu** [derekhe/msfs2020-google-map#43] [Q&A]
Thanks for the great application.
By the way, in the google maps data replacement.
Is there an option to get the zoom level of the image data to be retrieved?
It would be nice to have a more precise representation of the data when flying very low.
Thanks 
tarukuu

**Comment by derekhe** [comment]
I tried the high lod, which my combine underlaying pictures of next zoom level, can you tell me the difference?
![FlightSimulator_1e9UtUKzck](https://user-images.githubusercontent.com/1734077/140615468-7cb31859-a4a6-45bc-9713-31870e42ee8e.jpg)
![FlightSimulator_x9ehs5CFhS](https://user-images.githubusercontent.com/1734077/140615469-dd4bf994-b152-4879-b13d-cb41781594b3.jpg)

**Comment by tarukuu** [comment]
Thank you for your email.
I trying your tool.
Please you check.

https://youtu.be/DnSLQp3uSIA

I think your tool is very cool. Always I used. Thank you.

But a little I can not understand.

Please check my YouTube.

Thank you.

**Comment by tarukuu** [comment]
https://youtu.be/DnSLQp3uSlA
Sorry ↑

**Comment by derekhe** [comment]
What do you not understand?
